nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/Loggable.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.runtime.test.runner;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Annotation that will help you managing the logging when dealing with tests. You can create or update a logger
+ * dynamically by annotating your class or method test.
+ *
+ * @since 11.1
+ */
+@Inherited
+@Repeatable(Loggables.class)
+@Retention(RetentionPolicy.RUNTIME)
+@java.lang.annotation.Target({ ElementType.TYPE, ElementType.METHOD })
+
+public @interface Loggable {
```

Comment:
I found `Loggable` not really clear, what do you think about `LoggerLevel`?

Reply:
:), `LoggerLevel` was the initially name but i told my self perhaps there another better, then i remembered that the Spring Framework use this name, i told my self why not. But i think it's better to changes and mainly if in the futur we use Spring to avoid any crash types names